hudi-timeline-service/src/main/java/org/apache/hudi/timeline/service/FileSystemViewHandler.java

```diff
@@ -130,26 +147,54 @@ private boolean syncIfLocalViewBehind(Context ctx) {
   }
 
   private void writeValueAsString(Context ctx, Object obj) throws JsonProcessingException {
```

Comment:
can we have a top level `writeValueAsString` which does something like 

```
if (async) {
  writeValueAsStringAsync(ctx, obj)
} else {
   writeValueAsStringSync(ctx, obj)
}

```

instead of overloading the existing method to do sync and also branching for async?

Reply:
Done